apps/web-giddh/src/app/userDetails/userDetails.module.ts

```diff
@@ -60,6 +61,7 @@ const DEFAULT_PERFECT_SCROLLBAR_CONFIG: PerfectScrollbarConfigInterface = {
         DecimalDigitsModule,
         SharedModule,
         ElementViewChildModule,
+        AccordionModule.forRoot(),
```

Comment:
If we are not using accordian, remove this.

Reply:
fixed